PluginLoadError: failed to load plugin module defaults

The same error message is given for any of the commands

```
ric@ric-mbp repobee % repobee verify-settings
[ERROR] PluginLoadError: failed to load plugin module defaults
[ERROR] The plugin may not be installed, or it may not exist. If the plugin is defined in the config file, try running `repobee --no-plugins config-wizard` to remove any offending plugins.
```

Using v2.4.0 on Mac OS 10.15.4

--- a/src/_repobee/plugin.py
+++ b/src/_repobee/plugin.py
@@ -123,7 +123,8 @@ def _try_load_module(qualname: str) -> Optional[ModuleType]:
     """
     try:
         return importlib.import_module(qualname)
-    except ModuleNotFoundError:
+    except ModuleNotFoundError as exc:
+        plug.log.debug(str(exc))
         return None
 
 

